src/app/_components/BlockList/LayoutA/__tests__/__snapshots__/BlockListWithControls.test.tsx.snap

```diff
@@ -209,7 +209,7 @@ exports[`BlockListWithControls renders correctly 1`] = `
                   <span
                     class="chakra-text css-nvxyq8"
                   >
-                    in 54071 years
+                    in 54085 years
```

Comment:
Can we automate this such that we don't have to keep updating the tests

Reply:
I'd do this later as we need to fix the regression ASAP.